test-data/unit/check-modules.test

```diff
@@ -2495,3 +2495,6 @@ def __getattr__(attr: str) -> Any: ...
 class Cls: ...
 [builtins fixtures/module.pyi]
 [out]
+
+[case testFunctionWithReversibleDunderName]
+def __rpow__(self) -> int: ...
```

Comment:
While we're at it, maybe leave a test case checking to see what happens if we make this function an overload? And what happens if we define a top-level in-place operator method (like `__iadd__`)?

Reply:
Good idea. I will add tests for `__add__`, `__radd__`, and `__iadd__`, so we have covered all three cases.